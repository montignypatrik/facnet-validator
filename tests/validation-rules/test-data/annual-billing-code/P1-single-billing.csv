#;Facture;ID RAMQ;Date de Service;Début;Fin;Periode;Lieu de pratique;Secteur d'activité;Diagnostic;Code;Unités;Rôle;Élément de contexte;Montant Preliminaire;Montant payé;Doctor Info;DEV NOTE - À TRADUIRE: Report_DailyClaimInfo_TotalPrelAmount;DEV NOTE - À TRADUIRE: Report_DailyClaimInfo_TotalFinalAmount;DEV NOTE - À TRADUIRE: Report_DailyClaimInfo_TotalCount;Agence;Patient;Grand Total
1;128357270;15600245854;2025-03-15;08:00;;202520;55489;Aucun Secteur;;15815;0;1;;49,15;49,15;1069491-00000 | Dupont, Marie - Omnipraticien  ;49,15;49,15;1;;ABCD12345678 - Tremblay, Jean;
