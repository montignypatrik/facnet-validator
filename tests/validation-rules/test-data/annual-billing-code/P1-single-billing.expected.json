{
  "scenarioId": "P1",
  "scenarioName": "Single Billing Per Year",
  "description": "Code annuel facturé une seule fois pour un patient dans l'année civile",
  "expectedResults": [
    {
      "severity": "info",
      "category": "annual_limit",
      "message": "Validation réussie: Code annuel 15815 facturé correctement (1 fois) pour le patient en 2025",
      "solution": null,
      "ruleData": {
        "monetaryImpact": 0,
        "code": "15815",
        "patient": "ABCD12345678",
        "year": 2025,
        "totalCount": 1,
        "paidCount": 1,
        "unpaidCount": 0,
        "date": "2025-03-15",
        "amount": 49.15
      },
      "affectedRecordCount": 1
    },
    {
      "severity": "info",
      "category": "annual_limit",
      "message": "Validation complétée pour 1 enregistrement(s) de codes annuels. 0 erreur(s) détectée(s).",
      "solution": null,
      "ruleData": {
        "monetaryImpact": 0,
        "totalRecords": 1,
        "errorCount": 0,
        "totalAtRisk": 0,
        "codesChecked": ["15815"],
        "breakdown": {
          "E1": 0,
          "E2": 0,
          "E3": 0
        }
      }
    }
  ]
}
